### PR TITLE
IMP: expose --p-partree to align_to_tree_mafft_fasttree pipeline

### PR DESCRIPTION
**Pull request by mikerobeson**
**Improvement Description**
Exposes the `partree` parameter in this pipeline as outlined in #66. This will enable this pipeline to be used on very large sets of sequences. 

**References**
forum [xref](https://forum.qiime2.org/t/no-such-option-parttree/12916?u=nicholas_bokulich)

This has already been implemented for raxml as per #65.

### CODE REVIEW

**Comment by thermokarst** [comment]
Oops! Looks like the commits in this PR are a no-op:

Screen Shot 2020-02-14 at 1 13 49 PM

If you get a chance to push the changes up we still have time to get this in by EOB Monday for the 2020.2 release cycle, otherwise, let's push to 2020.5. Thanks!

**Comment by mikerobeson** [comment]
Hmm, guess I never actually pushed the commits... Oy vey!

File: q2_phylogeny/plugin_setup.py [hunk 1]
```diff
@@ -453,7 +454,9 @@
                                   'contains at least one character that is '
                                   'present in at least 40% of the sequences. '
                                   'This value is used when masking the '
-                                  'aligned sequences.'
+                                  'aligned sequences.',
+        'parttree': 'This flag is required if the number of sequences being '
+                    'aligned are larger than 1000000. Disabled by default. ',
```
Comment (thermokarst) [inline q2_phylogeny/plugin_setup.py]:
```suggestion
                    'aligned are larger than 1000000. Disabled by default.',
```

**Comment by thermokarst** [comment]
Thanks @mikerobeson - can you update the changelog on the forum when you get a chance?

**Pull request merged**